### PR TITLE
chore: testnet integration test

### PR DESCRIPTION
**Pull request by yehjxraymond**

### CODE REVIEW

File: scripts/integration.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+account_key=0xe82294532bcfcd8e0763ee5cef194f36f00396be59b94fb418f5f8d83140d9a7
+
+# Wait for ganache to listen on 8545
+sleep 2
+
+# Deploy document store to 0x63A223E025256790E88778a01f480eBA77731D04
+./node_modules/@govtechsg/open-attestation-cli/dist/cjs/index.js deploy document-store "My Document Store" -n local -k $account_key
```
Comment (rjchow) [inline scripts/integration.sh:7]: this looks a bit strange, no way to call it directly? or could use npx

File: src/services/publishing/publishing.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ const assertAddressIsSmartContract = async (address: string, wallet: Wallet): Pr
 export const publishVerifiableDocumentJob = async (
   job: PublishingJob,
   wallet: Wallet
-): Promise<string> => {
+): Promise<string | undefined> => {
```
Comment (rjchow) [inline src/services/publishing/publishing.ts:16]: should it throw an error instead if its undefined?

File: scripts/integration.sh (hunk 1 above)
Comment (Nebulis) [reply scripts/integration.sh:7]: there is the .bin folder too (node_modules/,bin/oa)

Comment (yehjxraymond) [reply scripts/integration.sh:7]: This is the script in package.json.

**Pull request merged**